File: ce/snippets/csharp/CRMV8/activityfeeds/cs/retrievepersonalwall.cs
// <snippetretrievepersonalwall>


 private void DisplayPersonalWallPage(int pageNumber)
 {
     // Retrieve the page of posts.  We'll only retrieve 5 at a time so that
     // we will have more than one page.
     var pageSize = 5;

     var personalWallPageReq = new RetrievePersonalWallRequest
     {
         CommentsPerPost = 2,
         PageNumber = pageNumber,
         PageSize = pageSize
     };
     var personalWallPageRes =
         (RetrievePersonalWallResponse)_serviceProxy.Execute(personalWallPageReq);

     Console.WriteLine("\r\n  Personal Wall Page {0} Posts:", pageNumber);
     foreach (Post post in personalWallPageRes.EntityCollection.Entities)
     {
         DisplayPost(post);
     }
 }

// </snippetretrievepersonalwall>

File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/activityfeedsvb/vb/retrievepersonalwall.vb
' <snippetretrievepersonalwall>



 Private Sub DisplayPersonalWallPage(ByVal pageNumber As Integer)
     ' Retrieve the page of posts.  We'll only retrieve 5 at a time so that
     ' we will have more than one page.
     Dim pageSize = 5

     Dim personalWallPageReq = New RetrievePersonalWallRequest With
                               {
                                   .CommentsPerPost = 2,
                                   .PageNumber = pageNumber,
                                   .PageSize = pageSize
                               }
     Dim personalWallPageRes = CType(_serviceProxy.Execute(personalWallPageReq), 
         RetrievePersonalWallResponse)

     Console.WriteLine(vbCrLf &amp; "  Personal Wall Page {0} Posts:", pageNumber)
     For Each post As Post In personalWallPageRes.EntityCollection.Entities
         DisplayPost(post)
     Next post
 End Sub

' </snippetretrievepersonalwall>